src/validation.h

```diff
@@ -859,9 +857,6 @@ class CChainState
         size_t max_coins_cache_size_bytes,
         size_t max_mempool_size_bytes) EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
 
-    /** Return list of MN EHF signals for current Tip() */
-    std::unordered_map<uint8_t, int> GetMNHFSignalsStage(const CBlockIndex* pindex);
```

Comment:
Instead removing it is possible to move this method from `CChainState` to `CChainStateHelper`. 

If do that, BlockAssembler can call `chainstate_helper->GeetMNHFSignalsStage` and no need to pass `mnhfManager` in constructor of BlockAssembler.